saleor/core/tasks.py

```diff
@@ -1,11 +1,13 @@
 from botocore.exceptions import ClientError
 from django.conf import settings
 from django.core.files.storage import default_storage
-from django.db.models import Exists, OuterRef, Q
+from django.db.models import Exists, OuterRef
 from django.utils import timezone
 
 from ..celeryconf import app
-from .models import EventDelivery, EventDeliveryAttempt, EventPayload
+from .models import EventDelivery, EventPayload
+
+BATCH_SIZE = 1000
```

Comment:
We should move `PERFORMANCE NOTE:` here.

Reply:
I've added comment